### PR TITLE
update architecture and polish README

### PR DESCRIPTION
**Pull request by stevemart**
* updated architecture from code pattern site
* use markdown instead of HTML so images don't look warped
* remove comments from the template
* add code styling

### CODE REVIEW

**Comment by stevemart** [comment]
@loafyloaf @kkbankol-ibm please review

File: README.md [hunk 1]
```diff
@@ -502,58 +393,34 @@ Click **Clear** on the Request Payload header to remove all payloads from the di
...
 # Troubleshooting
 
 * `sendPeersProposal - Promise is rejected: Error: 2 UNKNOWN: chaincode error (status: 500, message: Authorization for GETINSTALLEDCHAINCODES on channel getinstalledchaincodes has been denied with error Failed verifying that proposal's creator satisfies local MSP principal during channelless check policy with policy [Admins]: [This identity is not an admin]`
-> This error occurs if the certificate generated by the SDK user has not been uploaded to the peer
+> This error occurs if the certificate generated by the SDK user has not been uploaded to the peer.
 
 * `Error: The gRPC binary module was not installed. This may be fixed by running "npm rebuild"`
-> `grpc` is a requirement for the fabric-client SDK. Confirm that is has been installed in the `react_backend` directory with `npm install grpc@1.11.0`
-
-<!-- * Error: Environment {GUID} is still not active, retry once status is active
```
Comment (stevemart) [inline README.md]: Note to reviewers: This was commented out and from the template

File: README.md [hunk 2]
```diff
@@ -333,150 +288,86 @@ Finally, compile the `bundle.js` file
...
 The credentials for IBM Cloud services (Blockchain, Watson IoT Platform), can be found in the ``Services`` menu in IBM Cloud by selecting the ``Service Credentials`` option for each service.
 
-The Blockchain credentials consist of the `key`, `secret`, and `network_id` parameters
-<!-- ![]("https://i.imgur.com/Qof7sve.png" width="250" height="400") -->
-<p align="center">
-<img src="https://i.imgur.com/Qof7sve.png"  data-canonical-src="https://i.imgur.com/Qof7sve.png" width="450" height="450" style="margin-left: auto; margin-right: auto;">
-</p>
-
-These credentials will need to be provided to the UI in the next step
-<!-- We can obtain the Blockchain credentials by downloading
```
Comment (stevemart) [inline README.md]: Note to reviewers: This was commented out and from the template

File: README.md [hunk 3]
```diff
@@ -197,51 +178,42 @@ We can start by deploying an instance of the Watson IoT Service. In this pattern
...
-<img src="https://i.imgur.com/A2A6yXW.png" width="650" height="450">
-</p>
+Now, MQTT publish commands can be made from a device in the following format:
 
-Now, MQTT publish commands can be made from a device in the following format </br>
-Client ID: `d:${organization_id}:${device_type}:${device_id}` </br>
-Username: `use-token-auth` </br>
-Password: `${authentication_token}` </br>
-Endpoint: `${organization_id}.messaging.internetofthings.ibmcloud.com` </br>
+* Client ID: `d:${organization_id}:${device_type}:${device_id}`
```
Comment (stevemart) [inline README.md]: Made this bullets, because purdy

File: README.md [hunk 4]
```diff
@@ -16,60 +11,46 @@ When the reader has completed this Code Pattern, they will understand how to:
...
+5. The signed transaction is forwarded to an ordering service, which executes the transactions and places the result into the ledger.
 
-<!-- TODO expand on this -->
+6. The Monitoring UI auto-refreshes to show the transaction result and updated ledger in the "Response Payloads" and "Blockchain" columns, respectively.
 
 ## Install Prerequisites:
 ### IBM Cloud CLI
-To interact with the hosted offerings, the IBM Cloud CLI will need to be installed beforehand. The latest CLI releases can be found at the link [here](https://console.bluemix.net/docs/cli/reference/bluemix_cli/download_cli.html#download_install). An install script is maintained at the mentioned link, which can be executed with one of the following commands
+To interact with the hosted offerings, the IBM Cloud CLI will need to be installed beforehand. The latest CLI releases can be found at the link [here](https://console.bluemix.net/docs/cli/reference/bluemix_cli/download_cli.html#download_install). An install script is maintained at the mentioned link, which can be executed with one of the following commands.
```
Comment (loafyloaf) [inline README.md]: this redirects to https://console.bluemix.net/docs/cli/reference/ibmcloud/download_cli.html#download_install

File: README.md [hunk 5]
```diff
@@ -151,43 +130,45 @@ There are two methods we can use to deploy the application, either use the ``Dep
...
 
-1. To deploy the application to IBM Cloud, we'll need to leverage the IBM Cloud CLI. Ensure the cli is installed using the prerequisites section above, and then run the following command to deploy the application
-```
+1. To deploy the application to IBM Cloud, we'll need to leverage the IBM Cloud CLI. Ensure the cli is installed using the prerequisites section above, and then run the following command to deploy the application.
+
+```bash
 bx cf push
 ```
 
 2. To see the app and services created and configured for this Code Pattern, use the IBM Cloud dashboard, or run `bx cf apps` and `bx cf services` in the terminal. The app should be named `monitoring-ui` with a unique suffix.
```
Comment (loafyloaf) [inline README.md]: 'ibmcloud' instead of 'bx'?

**Comment by stevemart** [comment]
@loafyloaf addressed

**Pull request merged**